release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -2815,6 +2815,8 @@ This update contains changes from Kubernetes 1.22.6 up to 1.22.8. More informati
 
 * Previously, the *Git import* page failed to load due to performance improvement releases. With this fix, the *Git import* page no longer fails to load and now runs as intended. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2069621#[*BZ#2069621*])
 
+* Before this update, when users would provide a secret for a private repository in the *Import from Git* form, the secret was not decoded. As a result, the *Import from Git* form failed to detect the correct import strategy and builder image for the repository.  With this update, the *Import from Git* form decodes the secret before its use, which helps detecting the correct import strategy and builder image. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2069258[*BZ#2069258*])
```

Comment:
Rolfe also looked at this PR, and he had some good suggestions. The IBM SG wants us to use `would` as a hypothesis, or an unfulfilled condition. 

What do you think of Rolfe's suggested rewrites for the following?: 

 `if users provided a secret` instead of "when users would provide a secret" 

`which helps to detect OR which helps detect` instead of "which helps detecting"

Reply:
Thank you @jc-berger for this suggestion. I have incorporated the same. Please verify and confirm.